docs/source/markdown/podman-container-checkpoint.1.md

```diff
@@ -28,6 +28,60 @@ archives. Not compressing the checkpoint archive can result in faster checkpoint
 archive creation.\
 The default is **zstd**.
 
+#### **--create-image**=*image*
+
+Create a checkpoint image from the provided container. A checkpoint image is a
+standard OCI image with single layer (tar file) that contains all checkpoint
+files. The content of the image is in the same format as tar.gz file created
```

Comment:
This reads like the image would be stored as a tar.gz file.  But isn't the image created in the local image store?

Reply:
Yes, you are right. The image is created in the local image store. I will try to rephrase this.